FireRender.Maya.Src/AlembicWrapper.cpp

```diff
@@ -0,0 +1,889 @@
+
```

Comment:
please move this file to separate directory. Would it be reasonable to move it to SharedComponents ?